firmware/application/ui_navigation.cpp

```diff
@@ -165,6 +165,9 @@ SystemStatusView::SystemStatusView(
 
     rtc_battery_workaround();
 
+    // force apply of selected sdcard speed override at UI startup
+    pmem::set_config_sdcard_high_speed_io(pmem::config_sdcard_high_speed_io());
```

Comment:
It seems like this line should be _after_ loading persistent settings from the SD card (a few lines below).  Otherwise it seems it wouldn't be possible to enable high speed mode when the coin battery is dead.

Reply:
True, it would not restore at boot on pmem to sdcard conditions.
Changed order, thanks :-)